stable/weave-cloud/templates/manifest.yaml

```diff
@@ -0,0 +1,402 @@
+apiVersion: v1
+kind: List
```

Comment:
As I said, this is a copy of manifest generated by our API, so it would be very impractical for us to split this up into very small pieces, but we can split them up into 3 pieces (one per agent, i.e. Flux, Scope & Cortex), if that would help with review...